Vizzuality/half-earth-v3
# Feature/20 places data view

## Show 20 places in NRC, AOI, and Data view
### Description
- This PR shows the NRC future places layer, making it clickable so we can go to the future places AOI. Also there we can select different AOIs
- Also, on the Data view AOI section we have a new element on the dropdown for future places and you are able to go to the AOI from there.
- Also, Some refactoring of the AOI scene functions
### Designs
https://www.figma.com/file/2YAp3AS5lYCj2IWEAN8AJw/HE---Map?node-id=270%3A68878
### Testing instructions
- Go to the NRC page, select a country, click on one future places Aoi, a tooltip should open and when clicking on the action the AOI page should load correctly with all attributes
- On one places AOI page you should be able to click on other Future places shapes to open the tooltip and go to them
- On the main data view page, go to the AOI section, on the dropdown you should see future places, if you select it you should see the future places layer and be able to click on one of them to display a tooltip and go to the AOI
### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-298?atlOrigin=eyJpIjoiNmQ1MzVjNmY5NmVjNDIxY2EwZTQzODkzNjMyYzliZWYiLCJwIjoiaiJ9
https://vizzuality.atlassian.net/browse/HE-296?atlOrigin=eyJpIjoiYzczNzVkOWYwZjEzNDI4Yzg4NzAyYzJlNjRmNGJiOGEiLCJwIjoiaiJ9
https://vizzuality.atlassian.net/browse/HE-297?atlOrigin=eyJpIjoiMzViYWU4YjUyYmJmNDEzZmE0YjVhOWFiNjkwZDEyZTAiLCJwIjoiaiJ9
